### PR TITLE
Configure datasources and dashboards

### PR DESCRIPTION
**Pull request by hyvs**

### CODE REVIEW

**Comment by Tom32i** [comment]
:pray:

File: tasks/main.yml [hunk 1]
```diff
@@ -3,3 +3,7 @@
 - include: install.yml
 - include: config.yml
 - include: services.yml
+- include: datasources.yml
+  when: elao_grafana_datasources|length > 0
+- include: dashboards.yml
+  when: elao_grafana_dashboards|length > 0
```
Comment (benji07) [inline tasks/main.yml]: you forgot one line here

File: README.md [hunk 2]
```diff
@@ -56,6 +61,33 @@ elao_grafana_config:
     - admin_password: admin
 ```
 
+Configure datasources :
+
+```yaml
+elao_grafana_datasources:
+  - name: influxdb
+    type: influxdb
+    isDefault: true
```
Comment (benji07) [inline README.md:70]: underscore or camelCase ?

Comment (hyvs) [inline README.md:70]: I need to follow Grafana API naming -> camelCase
See http://docs.grafana.org/reference/http_api/#get-all-datasources

Comment (benji07) [inline README.md:70]: ok, I saw the `item|to_json`

**Pull request closed without merging**